pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -138,6 +142,9 @@ type VirtualCenterConfig struct {
 	// when ReloadVCConfigForNewClient is set to true it forces re-read config secret when
 	// new vc client needs to be created
 	ReloadVCConfigForNewClient bool
+
+	// session identifier
+	// SessionIdentifier string
```

Comment:
Is this supposed to be uncommented or removed?

Reply:
fixed in latest commit